packages/dapp/components/bonds/lib/hooks/useLaunchPartyContracts.ts

```diff
@@ -7,8 +7,8 @@ import {
   getChainlinkPriceFeedContract,
   getERC20Contract,
   getSimpleBondContract,
-  getUbiquiStickContract,
-  getUbiquiStickSaleContract,
+  getUbiquityStickContract,
+  getUbiquityStickSaleContract,
```

Comment:
I presume that the renames were undone for the UbiquiStick related code in this PR